_pages/laptop-setup.md

```diff
@@ -10,7 +10,7 @@ While you are welcome to customize your laptop, here are some tools that have wo
 * [chruby] for managing [Ruby] versions. 
 * [Cloud Foundry CLI] for command line access to 18F's Cloud Foundry-based application platform
 * [Docker] for all your containerization needs
-* [git-seekret] for preventing you from committing passwords and other sensitive information to a git repository
+* [Caulking] checks for many common types of API tokens and other sensitive information before you commit 
```

Comment:
Anywhere you mention sensitive information, I would link back to the Handbook page to reinforce that the Handbook is the canonical source of policy and guidance for protecting sensitive information.
```suggestion
* [Caulking] checks for many common types of API tokens and other [sensitive information](https://handbook.tts.gsa.gov/sensitive-information/) before you commit 
```